tests/util/test_file.cpp

```diff
@@ -95,9 +95,10 @@ SyncTestFile::SyncTestFile(const SyncConfig& sync_config)
     schema_mode = SchemaMode::Additive;
 }
 
-SyncTestFile::SyncTestFile(SyncServer& server)
+SyncTestFile::SyncTestFile(SyncServer& server, std::string name)
```

Comment:
This is used in the test that verifies that default values are merged correctly.